### PR TITLE
Api: Fetch all interactions for a fingerprint

### PR DESCRIPTION
**Pull request by Elisiac**
This will allow to pull all interactions regarding an specific event for processing outside of Comet.

### CODE REVIEW

File: comet_core/api_v0.py [hunk 1]
```diff
@@ -107,33 +107,36 @@ def assert_fingerprint_syntax(fingerprint):
...
+    Args:
+        validate_token (bool): A parameter to validate the request token or not
     Raises:
         ValueError: if the POST request did not contain json data,
         or if the json data did not contain a fingerprint
 
     Returns:
         str: fingerprint
     """
+    fingerprint = None
```
Comment (vikahl) [inline comet_core/api_v0.py]: Why setting the fingerprint to `None` here?

We will break out from the POST-part if fingerprint not in request_json by raising ValueError and (although less clear) from the GET part by the KeyError.

File: comet_core/api_v0.py [hunk 2]
```diff
@@ -467,3 +486,27 @@ def get_issues():
         return jsonify({"status": "error", "msg": "hydrate_open_issues failed"}), 500
 
     return jsonify(hydrated_issues)
+
+
+@bp.route("/interactions", methods=["GET"])
+def get_interactions_get():
+    """This endpoint expose the get_interactions functionality via GET request.
+    Doesn't required authentication as the information returned is deemed not sensitive.
```
Comment (vikahl) [inline comet_core/api_v0.py]: Good clarification in the comment! 👍

File: setup.py [hunk 3]
```diff
@@ -12,7 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
 import setuptools
 
 setuptools.setup(
```
Comment (vikahl) [inline setup.py]: Can you bump version to 2.4.0?

It's currently not handled automatically on merges :/

File: comet_core/api_v0.py (hunk 1 above)
Comment (Elisiac) [reply comet_core/api_v0.py]: Pycharm complains that the value might not be initialized. This makes sure whatever happens, there is a proper return value.

File: setup.py (hunk 3 above)
Comment (Elisiac) [reply setup.py]: Pushing the update now

**Pull request merged**